moveit_ros/planning/trajectory_execution_manager/test/test_execution_manager.test

```diff
@@ -1,20 +1,37 @@
-<?xml version="1.0"?>
 <launch>
-  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
-  <include file="$(find moveit_resources)/pr2_moveit_config/launch/planning_context.launch">
-    <arg name="load_robot_description" value="true"/>
-  </include>
+    <!-- Specify the config files to use -->
+    <rosparam ns="test_execution_manager" command="load" file="$(find moveit_ros_planning)/trajectory_execution_manager/test/config/moveit_cpp.yaml" />
 
-  <!-- If needed, broadcast static tf for robot root -->
-  <node pkg="tf2_ros" type="static_transform_publisher" name="virtual_joint_broadcaster_1" args="0 0 0 0 0 0 world base_footprint" />
+    <!-- Planning Pipeline -->
```

Comment:
Have a look at [`test_environment.launch`](https://github.com/ros-planning/moveit_resources/blob/100bb2dfa8d92832f1b06f45bfc6e8cded26cb75/panda_moveit_config/launch/test_environment.launch#L1). This should launch all required stuff.

Reply:
I wrote the tests with `moveit_cpp` so the `test_environment.launch` does not work for that. I could create the configuration in the `panda_moveit_config` package instead.

Reply:
You are right. You don't need a _move_group_ node when using MoveItCpp.
However, it doesn't harm either. Using `test_environment.launch` avoids all the boiler-plate code...